styles.css

```diff
@@ -50,9 +83,87 @@ p {
   font-size: 16px;
   line-height: 24px;
   color: #fff;
+  padding-bottom: 10px;
+}
+
+.titles2 {
+  font-family: 'Roboto', sans-serif;
+  font-style: normal;
+  color: #fff5e1;
+  padding-bottom: 5px;
+}
+
+.centrar {
+  font-family: 'Roboto', sans-serif;
+  font-style: normal;
+  color: #fff5e1;
+  padding-bottom: 5;
+}
+
+.centrar1 {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
```

Comment:
Kindly fix your design to match the Figma template design 
![image](https://user-images.githubusercontent.com/66112760/164884825-2efa3b5e-2f78-462e-ab29-b8a8a9aac324.png)